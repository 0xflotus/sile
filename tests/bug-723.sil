\begin[papersize=a6]{document}
\nofolios
\neverindent
\pagetemplate[first-content-frame=ltr]{
\frame[id=ltr,direction=LTR-TTB,top=top(content),left=left(content),right=right(content),height=25%ph,next=rtl]
\frame[id=rtl,direction=RTL-TTB,top=bottom(ltr),left=left(content),right=right(content),height=25%ph,next=ttb]
\frame[id=ttb,direction=TTB-LTR,top=bottom(rtl),left=left(content),right=50%pw,bottom=bottom(content),next=btt]
\frame[id=btt,direction=BTT-RTL,top=bottom(rtl),left=right(ttb),right=right(content),bottom=bottom(content)]
}
\script[src=packages/rules]
Hello \hrule[width=30%pw,height=1ex] World\par
\framebreak

\font[family=LateefGR,language=ar]
مرحبا \hrule[width=30%pw,height=1ex] بالعالم\par
\framebreak

\font[family=Mongolian White,language=mn]
ᠡᠷᠲᠡ \hrule[width=30%pw,height=1ex] ᠤᠷᠢᠳᠠ\par
\framebreak

foo \hrule[width=30%pw,height=1ex] bar\par
\end{document}
