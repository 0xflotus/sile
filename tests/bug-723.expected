Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	28.5447
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 72 79 79 82	(Hello)
Draw line	39.1448 24.0037 89.2913 4.5410
Mx 	131.0926
T	58 82 85 79 71	(World)
Mx 	269.8750
My 	131.3612
Set font 	LateefGR;10;400;;normal;;RTL
T	472 669 811 506 885	(مرحبا)
Draw line	267.1341 127.9921 -89.2913 3.3691
Mx 	159.0764
T	631 879 472 720 879 472 792	(بالعالم)
Mx 	24.8819
My 	230.7402
Set font 	Mongolian White;10;400;;normal;;LTR
T	225 678 636 230	(ᠡᠷᠲᠡ)
Draw line	34.8819 238.2135 -10.0000 89.2913
My 	330.8327
T	262 678 243 648 216	(ᠤᠷᠢᠳᠠ)
Mx 	275.7637
My 	377.5748
T	68 91 91	(foo)
Draw line	280.2364 361.0167 -4.4727 -89.2913
My 	268.3996
T	40 32 113	(bar)
End page
Finish
